### PR TITLE
feat: add address consumer

### PR DESCRIPTION
**Pull request by emalfroy**

### CODE REVIEW

File: src/ParcelRegistry.Consumer.Address/Projections/AddressKafkaProjection.cs [hunk 1]
```diff
@@ -0,0 +1,141 @@
+namespace ParcelRegistry.Consumer.Address.Projections
+{
+    using System;
+    using Be.Vlaanderen.Basisregisters.GrAr.Contracts.AddressRegistry;
+    using Be.Vlaanderen.Basisregisters.ProjectionHandling.Connector;
+
+    public class AddressKafkaProjection : ConnectedProjection<ConsumerAddressContext>
+    {
+        public AddressKafkaProjection()
+        {
```
Comment (jvandaal) [inline src/ParcelRegistry.Consumer.Address/Projections/AddressKafkaProjection.cs:10]: We're missing: AddressWasDeregulated

File: src/ParcelRegistry.Consumer.Address/appsettings.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "ConnectionStrings": {
+    "ConsumerAddress": "Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory.AddressRegistry;Trusted_Connection=True;TrustServerCertificate=True;",
```
Comment (jvandaal) [inline src/ParcelRegistry.Consumer.Address/appsettings.json:3]: AddressRegistry -> ParcelRegistry

File: src/ParcelRegistry.Consumer.Address/Projections/AddressKafkaProjection.cs (hunk 1 above)
Comment (ArneD) [reply src/ParcelRegistry.Consumer.Address/Projections/AddressKafkaProjection.cs:10]: Also needed in building?

**Review by ArneD — APPROVED** [review]

**Pull request merged**